crates/modelardb_server/src/query/generated_as_exec.rs

```diff
@@ -195,6 +198,48 @@ impl GeneratedAsStream {
             baseline_metrics,
         }
     }
+
+    /// Format and return the first row in `batch` that causes `physical_expr` to fail, if
+    /// `physical_expr` never fails or if `batch` is from a normal table, [`None`] is returned.
+    fn failing_row(batch: &RecordBatch, physical_expr: &Arc<dyn PhysicalExpr>) -> Option<String> {
+        for row_index in 0..batch.num_rows() {
+            if physical_expr.evaluate(&batch.slice(row_index, 1)).is_err() {
```

Comment:
Can it happen that when the user SELECTs from a generated column it fails, but when we try here it does not? If RANDOM(), .e.g., is used in the expression that could potentially (although, unlikely) happen

Reply:
Yes that should be possible since we execute the physical expression again. One solution is to do like MySQL and just reject functions that can cause such problems: `ERROR 3763 (HY000): Expression of generated column 'generated' contains a disallowed function: rand.` However, this would require us to continuously maintain a list of functions that Apache Arrow DataFusion implements which can cause such problems and limits what users can do. So instead I have updated the function to return an error if the table is not a model table (i.e., we have a bug in the planner) and otherwise return the row as a string or the string "Unknown Row".